samples/BankAccount/BankClient/Program.cs

```diff
@@ -1,52 +1,64 @@
 using AccountTransfer.Interfaces;
-using Microsoft.Extensions.Logging;
-using Orleans;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
 
-using var client = new ClientBuilder()
-    .UseLocalhostClustering()
-    .ConfigureLogging(logging => logging.AddConsole())
+using IHost host = Host.CreateDefaultBuilder(args)
+    .UseOrleansClient(client =>
+    {
+        client.UseLocalhostClustering()
+            .UseTransactions();
+    })
     .Build();
 
-await client.Connect();
+await host.StartAsync();
```

Comment:
Should the host be stopped (explicitely) after the loop too?
More or less for completeness and to show good behavior.

Reply:
Yes, you're right, it should

Reply:
Actually, we should change this to use the host lifecycle (which should be the console lifecycle). I'll push an update.